cypress/e2e/course/contextRestrictionOfCtlTools.feature

```diff
@@ -140,18 +158,15 @@ Feature: Course - Restrict CTL tools to context course, board-element, media-boa
     Then I do not see tool 'CY Test Tool Board-Element Restriction' in the available media line
 
   @unstable_test
-  Scenario: Post-test: Teacher deletes course, admin deletes external tools
-    Given I am logged in as a 'teacher1_nbc' at 'nbc'
-    When I go to rooms overview
-    When I go to room 'Cypress Test Course'
-    When I open page Edit course
-    When I click on the button delete course
-    Then I see the modal to confirm the deletion
-    When I click on the button delete on the modal to confirm the course deletion
-    Then I do not see the course 'Cypress Test Course' on the room overview page
+  Scenario: Post-test: Admin deletes a course and external tools
+    Given I am logged in as a 'admin1_nbc' at 'nbc'
+    When I click on administration in menu
+    When I go to course administration page
+    When I click the delete button for course 'Cypress Test Course' in course table
+    Then I see the delete modal
+    When I click the confirmation button on the delete modal
```

Comment:
You should test of the course is deleted then, I think you deleted the THEN accidently